app/api/pose-estimation/pose-estimation.controller.js

```diff
@@ -185,10 +193,20 @@ exports.computePoseCreateGltf = route(async (req, res) => {
 
      // Compute surface covered with gcps
      const ratio = computeGCPRatio(GCPs, width, height);
-     filteredResults.gcpPercentSurface = ratio
+     filteredResults.gcpPercentSurface = ratio;
+
+     //Add region and url_gltf in results
+     filteredResults.image_id = id;
+     filteredResults.regionByPx = regionByPx;
 
+     //Build gltf_url
+     let region_url = "";
+      if (regionByPx) {
+        region_url = `_${regionByPx[0]}_${regionByPx[1]}_${regionByPx[2]}_${regionByPx[3]}`;
+      }
+     filteredResults.gltf_url = `${config.apiUrl}/data/collections/${collection_id}/gltf/${id}${region_url}.gltf`;
```

Comment:
probably we could refactor that logic in a util methods. Since used multiple time, it would ensure we do not have any typo when generating that url